Assets/MixedRealityToolkit.Examples/Demos/UX/BoundingBox/Scripts/BoundingBoxTest.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.MixedReality.Toolkit.UI
 {
-    public class BoundingBoxTest : InputSystemGlobalListener, IMixedRealitySpeechHandler
```

Comment:
Might be interesting to make `InputSystemGlobalListener<T>` to continue helping devs make global listeners, instead of needing to manually register and unregister.